question28/css/news.css

```diff
@@ -0,0 +1,164 @@
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+code,
+form,
+fieldset,
+legend,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+  margin: 0;
+  padding: 0;
+}
+.container {
+    max-width: 1000px;
+    width: 100%;
+    margin: 30px auto;
+}
+.logout_btn {
+    text-align: right;
+}
+.logout_btn button {
+    padding: 10px 30px;
+    margin-bottom: 30px;
+    background: #eeeeee;
+    cursor: pointer;
+}
+.newsWrapper {
+    /* margin: 50px auto; */
+}
+.news_tabs {
+    display: -webkit-flex;
+    display: flex;
+    justify-content: center;
+}
+.news_tabs li{
+    flex-grow: 1;
+    padding: 10px;
+    list-style: none;
+    border: solid 1px #CCC;
```

Comment:
```suggestion
    border: solid 1px #ccc;
```

Reply:
Thank you for your review! Fixed so please confirm.

fixed commit : https://github.com/csakyo/homework/pull/42/commits/e203f15d7089da4f50e76d8177d9fd75ef1ae938